models/index.go

```diff
@@ -34,6 +34,12 @@ func (m *IndexModel) Get() (currentAndNext *myradio.CurrentAndNext, banners []my
 		return
 	}
 
+	selectorInfo, err := m.session.GetSelectorInfo()
+	if err != nil {
+		return
+	}
+	jukebox = selectorInfo.Studio == myradio.SelectorJukebox
```

Comment:
Hmm, this might not account for off air? Probably need to add source 8 in myradio-go and then check against that also.